app/services/foreman/renderer/scope/macros/helpers.rb

```diff
@@ -108,6 +108,26 @@ def shell_escape(string)
           def current_date(format: '%F')
             Time.zone.today.strftime(format)
           end
+
+          apipie :method, 'Checks whether a value is truthly or not' do
```

Comment:
```suggestion
          apipie :method, 'Checks whether a value is truthy or not' do
```

so this matches the method name?